templates/head/vwo_test_script.twig

```diff
@@ -0,0 +1,12 @@
+<!-- Start VWO Async SmartCode -->
+<link rel="preconnect" href="https://dev.visualwebsiteoptimizer.com" />
+<script type='text/javascript' id='vwoCode'>
```

Comment:
Let's move the JS code out of the twig tempalte to be consistent with the @mardelnet's recent work at #2502